src/Console/Application.php

```diff
@@ -76,9 +76,7 @@ private function loadUserConfiguration(ContainerBuilder $container): void
         $configFile = $homeDirectory . '/.release-tool/auth.yml';
 
         if (!file_exists($configFile)) {
-            throw new RuntimeException(
-                sprintf('The file %s needs to exist and contain a GitHub access token', $configFile)
-            );
+            exit(sprintf('The file %s needs to exist and contain a GitHub access token.' . PHP_EOL, $configFile));
```

Comment:
This feels like a HTTP 200 OK response with a error message in the body. This error message is way better than a complete exception stack trace but we should not pretend that nothing happend and stop with a clean exit.

At least a exit(1) (https://tldp.org/LDP/abs/html/exitcodes.html) should be used to let the terminal, and user, know that an error has occurred.